reverse_replication/README.md

```diff
@@ -93,20 +93,38 @@ go run launcher.go -projectId=my-project -dataflowRegion=us-east1 -instanceId=my
+
+#### Recovery steps for the infinite loop
 1. The user must track the last handled window that was successfully processed - this can be obtained from the Dataflow logs or by checking the watermark of the job stage.
-2. Specify a different partition metadata database or drop the previous metadata tables  - since watermarks for a given partition are persisted and hence will be read from that point onwards. So the pipeline must begin with a clean partition metadata database.Note that the metadata tables can be dropped by giving DROP TABLE\<table name\> statements in the Cloud UI.
-3. The current pipeline must be updated - the user must specify the start time of change stream query as the last successfully processed timestamp and the end time of the change stream as the time that would result in ~1GB writes. Example, if average record size is 1KB, then 10,00,000 records should be processed in a window and if the TPS during that window was 20K, then the window must end at 50 second. Detailed steps to update the Dataflow job are given here: https://cloud.google.com/dataflow/docs/guides/updating-a-pipeline#gcloud-cli
-4. Once this window is processed, the pipeline must be updated with next set of start and end times, until the pipeline catches up and the finally the end timestamp need not be passed.
+2. Specify a different partition metadata database or drop the previous metadata tables - since watermarks for a given partition are persisted and hence will be read from that point onwards. So the pipeline must begin with a clean partition metadata database.Note that the metadata tables can be dropped by giving DROP TABLE\<table name\> statements in the Cloud UI.
+3. The current pipeline must be updated - the user must specify the start time of the change stream query as the last successfully processed timestamp and the end time of the change stream as the time that would result in ~1GB writes. Example, if the average record size is 1KB, then 10,00,000 records should be processed in a window and if the TPS during that window was 20K, then the window must end at 50 seconds. Detailed steps to update the Dataflow job are given here.
+4. Once this window is processed, the pipeline must be updated with the next set of start and end times, until the pipeline catches up and finally the end timestamp need not be passed.
+
+## Double Load on Spanner
+- The reverse replication pipeline in conjunction with the forward migration pipeline can cause double load on Spanner. Each write on Spanner gets reverse replicated to MySQL, which further gets forward replicated back onto Spanner. This can cause double the load and temporarily put Spanner in an inconsistent state, till the time the forward pipeline is deleted.
+- It is recommended to start the reverse replication pipeline just before cutover and stop the forward migration pipeline post cutover.
+
+## Error Scenarios
```

Comment:
Why is this a limitation? Note that it's covered in the user guide under the troubleshooting section.